aeron-client/src/main/c/uri/aeron_uri.h

```diff
@@ -75,6 +75,7 @@ aeron_uri_params_t;
 #define AERON_URI_NAK_DELAY_KEY "nak-delay"
 #define AERON_URI_UNTETHERED_WINDOW_LIMIT_TIMEOUT_KEY "untethered-window-limit-timeout"
 #define AERON_URI_UNTETHERED_RESTING_TIMEOUT_KEY "untethered-resting-timeout"
+#define AERON_URI_MAX_RETRANSMITS_KEY "retransmits-active-max"
```

Comment:
Variable name does not match the value. Other keys use `AERON_URI_VALUE_KEY "value"`.